data/languages.yaml

```diff
@@ -0,0 +1,1514 @@
+        - ต.ค.
+        - ตุลา
+    november:
+        - พฤศจิกายน
+        - เดือนพฤศจิกายน
+        - พ.ย.
+        - พฤศจิ
+    december:
+        - ธันวาคม
+        - เดือนธันวาคม
+        - ธ.ค
+        - ธันวา
+
+    year:
+        - ปี
+    month:
+        - เดือน
+    week:
+        - สัปดาห์
+    day:
+        - วัน
+    hour:
+        - ชั่วโมง
+        - ชม
+    minute:
+        - นาที
+    second:
+        - วินาที
+
+    ago:
+        - มาแล้ว
+        - ก่อน
+
+    simplifications:
+        - วันนี้: 0 วัน
+        - เมื่อวานนี้: 1 วัน
+        - 1 วันที่แล้ว: 1 วัน
+        - เมื่อวานซืน: 2 วัน
+        - 2 วันที่แล้ว: 2 วัน
+        - เมื่อสักครู่นี้: 0 วินาที
+
+pl:
+    name: Polish
+
+    skip: ['r.', 'roku', 'w', 'o']
+
+    monday:
+        - Poniedziałek
+        - Poniedzialek
+        - Pon
+        - Pn
+    tuesday:
+        - Wtorek
+        - Wt
+        - Wto
+    wednesday:
+        - Środa
+        - Sroda
+        - Śro
+        - Sro
+        - Śr
+        - Sr
+    thursday:
+        - Czwartek
+        - Czw
+        - Cz
+    friday:
+        - Piątek
+        - Piatek
+        - Pią
+        - Pia
+        - Pi
+        - Pt
+    saturday:
+        - Sobota
+        - Sob
+        - So
+        - Sb
+    sunday:
+        - Niedziela
+        - Nie
+        - Ni
+        - Nd
+
+    january:
+        - Styczeń
+        - Styczen
+        - Stycznia
+        - Sty
+    february:
+        - Luty
+        - Lutego
+        - Lut
+    march:
+        - Marzec
+        - Marca
+        - Mar
+    april:
+        - Kwiecień
+        - Kwiecien
+        - Kwietnia
+        - Kwi
+    may:
+        - Maj
+        - Maja
+    june:
+        - Czerwiec
+        - Czerwca
+        - Cze
+    july:
+        - Lipiec
+        - Lipca
+        - Lip
+    august:
+        - Sierpień
+        - Sierpien
+        - Sierpnia
+        - Sie
+    september:
+        - Wrzesień
+        - Wrzesien
+        - Września
+        - Wrzesnia
+        - Wrz
+    october:
+        - Październik
+        - Pazdziernik
+        - Października
+        - Pazdziernika
+        - Paź
+        - Paz
+    november:
+        - Listopad
+        - Listopada
+        - Lis
+    december:
+        - Grudzień
+        - Grudzien
+        - Grudnia
+        - Gru
+
+    year:
+        - rok
+        - lata
+        - lat
+    month:
+        - miesiąc
+        - miesiac
+        - miesięcy
+        - miesiąca
+    week:
+        - tydzień
+        - tydzien
+        - tygodni
+        - tygodnie
+    day:
+        - dzień
+        - dzien
+        - dnia
+        - dniu
+    hour:
+        - godzina
+        - godz.
+    minute:
+        - minuta
+        - min
+    second:
+        - drugi
+
+    ago:
+        - temu
+
+    simplifications:
+        - dziś: 0 dnia
+        - dzisiaj: 0 dnia
+        - onegdaj: 2 dnia
+        - przedwczoraj: 2 dnia
+        - wczoraj: 1 dnia
+
+fa:
+    name: Persian
+
+    skip: ['\u064e', '\u064f']
```

Comment:
Please avoid unicode escaping here. We want native speakers easily to read through the document